src/calva-fmt/src/extension.ts

```diff
@@ -82,4 +79,9 @@ export async function activate(context: vscode.ExtensionContext) {
     )
   );
   vscode.window.onDidChangeActiveTextEditor(inferer.updateState);
+  vscode.workspace.onDidChangeConfiguration((e) => {
+    if (e.affectsConfiguration('calva.fmt.formatAsYouType')) {
```

Comment:
Isn't this setting removed now?

Reply:
Might be a merge thing that I missed... I vaguely remember something with a strange merge when doing something recently.